app/editor/src/components/tool-bar/styled/ToolBarSection.tsx

```diff
@@ -48,4 +48,18 @@ export const ToolBarSection = styled(Col)`
     padding: 0.5em;
     justify-content: center;
   }
+  .title-container {
```

Comment:
Do you think this should be added to a separate styled component @Fosol? Or is that overkill - for example have it in `styled/ContentStatusSection.tsx` as it is specific to that component and not the general default `ToolBarSection`

Reply:
I think I covered this in my comment on the `ToolBarSection2.tsx` file.